docs/en/observability/monitor-infra/host-details-partial.asciidoc

```diff
@@ -52,18 +52,39 @@ along with the total number of processes in these various states:
 The processes listed in the *Top processes* table are based on an aggregation of the top CPU and the top memory consuming processes.
 The number of top processes is controlled by `process.include_top_n.by_cpu` and `process.include_top_n.by_memory`.
 
-|=== 
+|===
 
 | *Command* | Full command line that started the process, including the absolute path to the executable, and all the arguments (`system.process.cmdline`).
 | *PID* | Process id (`process.pid`).
 | *User* | User name (`user.name`).
 | *CPU* | The percentage of CPU time spent by the process since the last event (`system.process.cpu.total.pct`).
-| *Time* | The time the process started (`system.process.cpu.start_time`). 
-| *Memory* | The percentage of memory (`system.process.memory.rss.pct`) the process occupied in main memory (RAM). 
+| *Time* | The time the process started (`system.process.cpu.start_time`).
+| *Memory* | The percentage of memory (`system.process.memory.rss.pct`) the process occupied in main memory (RAM).
 | *State* | The current state of the process and the total number of processes (`system.process.state`). Expected values are: `running`, `sleeping`, `dead`, `stopped`,
 `idle`, `zombie`, and `unknown`.
 
-|=== 
+|===
+====
+
+[%collapsible]
+.*Universal Profiling*
+====
+
+[role="screenshot"]
+image::images/universal-profiling-overlay.png[Host Universal Profiling]
+
+The *Universal Profiling* tab shows CPU usage down to the application code level.
+From here, you can find the sources of resource usage, and identify code that can be optimized to reduce infrastructure costs.
+The Universal Profile tab has the following views.
+
+|===
+
+| *Flamegraph* | A visual representation of the functions that consume the most resources. Each rectangle represents a function. The rectangle width represents the time spent in the function. The number of stacked rectangles represents the stack depth, or the number of functions called to reach the current function.
+
+| *Top 10 Functions* | A list of the most expensive lines of code on your host. See the most frequently sampled functions, broken down by CPU time, annualized CO2, and annualized cost estimates.
```

Comment:
I don't know what annualized CO2 means. Will users? I just know it's not good for the planet. :-P